tfjs-backend-webgpu/src/webgpu_program.ts

```diff
@@ -19,6 +19,11 @@ import {backend_util, DataType, DataTypeMap, env, Rank, TensorInfo, util} from '
 
 import {symbolicallyComputeStrides} from './shader_util';
 
+export enum PixelsOpType {
+  FROM_PIXELS,
+  TO_PIXELS
```

Comment:
To keep consistent, should we rename `TO_PIXELS` to `DRAW`?

Reply:
Done, PTAL